tox.ini

```diff
@@ -100,14 +101,23 @@ commands =
         --cov-report=term-missing --cov-fail-under=100 --cov-config .coveragerc
 
 
+# Documentation build run, PDF target
+[testenv:pdfdoc]
+skip_install = True
+usedevelop = True
+deps = {[testenv]deps}
```

Comment:
I have two comments, not about this line, but in general about building the PDF documentation:

1. I had to install the `sphinxcontrib-spelling` module, otherwise it fails. I guess this needs to be added into `.virtualenv.dev-requirements.txt`?
1. Another alternative idea
   Would a separate requirement file under `kiwi/doc` be useful? Use case could be if someone is only interested in the documentation. With a separate file would that be possible. You can still refer to the "doc requirements" from  `.virtualenv.dev-requirements.txt` for a complete build.

Reply:
sphinxcontrib-spelling is listed in .virtualenv.dev-requirements.txt. That should be part of your tox env if you have set it up using this documentation ;)

Separation of the dev-requirements is currently not an option because I don't think we gain much value. The setup speed of a development env for kiwi is currently not really relevant. The components download and compiling is done in under ~2min. However if I separate them I need to change the tox env for building the docs or running the tool which I think is not handy